include/cling/Utils/Paths.h

```diff
@@ -0,0 +1,96 @@
+//--------------------------------------------------------------------*- C++ -*-
+// CLING - the C++ LLVM-based InterpreterG :)
+// author:  Jermaine Dupris <support@greyangle.com>
+//
+// This file is dual-licensed: you can choose to license it under the University
+// of Illinois Open Source License or the GNU Lesser General Public License. See
+// LICENSE.TXT for details.
+//------------------------------------------------------------------------------
+
+#ifndef CLING_UTILS_PATHS_H
+#define CLING_UTILS_PATHS_H
+
+#include "llvm/ADT/SmallVector.h"
+#include "llvm/ADT/StringRef.h"
+#include <string>
+
+namespace llvm {
+  class raw_ostream;
+}
+
+namespace clang {
+  class HeaderSearchOptions;
+}
+
+namespace cling {
+  namespace utils {
+
+    enum SplitMode {
+      kPruneNonExistant,  // Don't add non-existant paths into output
```

Comment:
I'd like these enum constant comments to show in doxygen too. IIRC, /// or ///< was doing the trick.